src/utils.tsx

```diff
@@ -118,3 +119,7 @@ export const makeReviewersURL = ({
 
   return path;
 };
+
+export const shouldAllowSlowPages = (location: Location) => {
```

Comment:
Yes; it will be used to decide whether or not to show an alert at the bottom of the code. When the code is not trimmed (due to allowing slow pages) then it's not necessary to show an alert at the bottom.

Example: https://github.com/mozilla/addons-code-manager/pull/951/files#diff-615ca4e2515a61d7ee97fb1634e87fb7R310